Makefile

```
@@ -17,9 +20,51 @@ define compile_service
+endef
+
 $(SERVICES):
 	$(call compile_service,$(@))
 
+$(DOCKERS):
+	$(call make_docker,$(@),$(GOARCH))
+
+$(DOCKERS_DEV):
+	$(call make_docker_dev,$(@))
+
+dockers: $(DOCKERS)
```

checkmake (0.2.2)

[warning] Target "dockers" should be declared PHONY.

(phonydeclared)